README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo
 
-[Live Demo Link](https://vanecode.github.io/art-gallery/)
+[Live Demo Link](https://vanecode.github.io/to-do-list/dist/)
 
 ## Getting Started
 
```

Comment:
- [ ] Your **Getting Started** section lets the other people know how to start your project on their machine. So, it would be very nice if you add the following commands in your **Getting Started** section.

# Commands
![getting-started-fixed](https://user-images.githubusercontent.com/93039359/179321996-c3e2d5fb-22ae-4e96-a1ad-a0f6938e124e.png)